Cargo.toml

```diff
@@ -165,8 +165,8 @@ rmp-serde = { version = "0.15.5", default-features = false, optional = true }
 rmpv = { version = "0.4.7", default-features = false, features = ["with-serde"], optional = true }
 
 # Prost
-prost = { version = "0.7.0", default-features = false, features = ["std"]  }
-prost-types = { version = "0.7.0", default-features = false }
+prost = { version = "0.8", default-features = false, features = ["std"]  }
+prost-types = { version = "0.8", default-features = false }
```

Comment:
I think you may find that we cannot actually upgrade yet. See https://github.com/timberio/vector/pull/8273 . The vector source/sink integration tests seem to hang with the upgrade. I haven't had a chance to poke at it yet though.

Reply:
Actually, I was mistaken. I was thinking of the Tokio upgrade PR #8151

Reply:
😓 so it was the unit tests that were hanging. This appears to still be the case with the upgrade: https://github.com/timberio/vector/pull/8273/checks?check_run_id=3194383862

Reply:
This should be resolved in https://github.com/timberio/vector/pull/8401/commits/c6d02bfa9561986174344dea8fe44bdb4cee4167.